test/parallel/test-assert.js

```diff
@@ -476,16 +476,16 @@ assert.throws(() => {
 {
   // Bad args to AssertionError constructor should throw TypeError.
   const args = [1, true, false, '', null, Infinity, Symbol('test'), undefined];
-  args.forEach((input) => {
+  for (const input of args) {
     assert.throws(
       () => new assert.AssertionError(input),
       {
         code: 'ERR_INVALID_ARG_TYPE',
         name: 'TypeError',
         message: 'The "options" argument must be of type object.' +
-                 common.invalidArgTypeHelper(input)
+        common.invalidArgTypeHelper(input)
```

Comment:
This change in indentation seems unrelated. Was it mandated by the linter?
```suggestion
                 common.invalidArgTypeHelper(input)
```

Reply:
I don't believe so. I think it was an accidental indent.